.clang-tidy

```diff
@@ -0,0 +1,73 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
[fix] Can we remove the duplicate license identifier

Reply:
Will do.  I wasn't sure how it should be listed, e.g. if each copyright needs to be paired with a license, but then I guess multiple copyrights only makes sense if they're all Apache 2?

Reply:
Sounds right - as you can't assume that you can mix licenses.